test-it/ExpectedResults/StringScanBoolean.vtr

```diff
@@ -0,0 +1,20 @@
+offset_past:<4>, remaining:()
+:<true> err:<(false 0 '')>
+offset_past:<1>, remaining:()
+:<true> err:<(false 0 '')>
+offset_past:<3>, remaining:()
+:<true> err:<(false 0 '')>
+offset_past:<2>, remaining:()
+:<true> err:<(false 0 '')>
+offset_past:<4>, remaining:()
+:<true> err:<(false 0 '')>
+offset_past:<5>, remaining:()
+:<false> err:<(false 0 '')>
+offset_past:<5>, remaining:()
+:<false> err:<(false 0 '')>
+offset_past:<2>, remaining:()
+:<false> err:<(false 0 '')>
+offset_past:<3>, remaining:()
+:<false> err:<(false 0 '')>
+offset_past:<5>, remaining:()
+:<false> err:<(false 0 '')>
```

Comment:
Add a test for correct behavior when an error occurs (the expected Boolean text isn't there, or end of string).